contracts/TokenPool.sol

```diff
@@ -121,31 +121,31 @@ contract TokenPool is Migratable, Pausable {
 
```

Comment:
In line 120, update error message to be different, or combine into a single require.

Reply:
message updated